app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt

```diff
@@ -65,7 +68,7 @@ fun SearchBarUI(placeholderText: String, modifier: Modifier = Modifier, onTextTy
                     }) {
                         Icon(
                             modifier = Modifier.height(35.dp),
-                            imageVector = Icons.Filled.Close,
+                            painter = painterResource(id = R.drawable.ic_clear_search),
```

Comment:
is this icon going to be specific for clearing the search only  ? or could it be reused ? if yes, maybe naming it to ic_clear would be better